src/routing/bootstrap.rs

```diff
@@ -213,7 +230,7 @@ impl<'a> State<'a> {
                             return Ok((response, sender))
                         }
                     }
-                }
+                },
```

Comment:
is this added by the cargo-fmt automatically?

Reply:
No. Don't know how it got there. I'll remove it, but unless there are other changes in this PR I'll do it in the next one.